[sngrep] Unable to build

Same issue for all architectures.
```
$ crew build sngrep
sngrep: An Ncurses SIP Messages flow viewer
https://github.com/irontec/sngrep
version 1.4.6
Downloading source...
################################################################################################################################################ 100.0%################################################################################################################################################ 100.0%
Archive downloaded
Unpacking archive using 'tar', this may take a while...
Building from source, this may take a while...
Rename all *.la files to *.la_tmp
Generating the configure script ...
sngrep failed to build: `./configure CPPFLAGS=-I/usr/local/include/ncursesw --prefix=/usr/local --libdir=/usr/local/lib64 --with-openssl --with-pcre --disable-logo --enable-ipv6` exited with 127
```

For whatever reason, I'm still unable to build on armv7l and i686.  See below:
```
checking for pcap_open_offline in -lpcap... no
configure: error:  You need to have libpcap installed to compile sngrep.
sngrep failed to build: `./configure CPPFLAGS="-I/usr/local/include/ncursesw -I/usr/local/include/libpcap"           --prefix=/usr/local           --libdir=/usr/local/lib           --with-openssl           --with-pcre           --disable-logo           --enable-ipv6` exited with 1
chronos@localhost ~/Downloads $ crew search libpcap
(i) libpcap: A portable C/C++ library for network traffic capture.
```
As you can see, the configure script cannot find libpcap even though the package is installed and the library path is included in the configure options.

--- a/packages/sngrep.rb
+++ b/packages/sngrep.rb
@@ -3,21 +3,29 @@ require 'package'
 class Sngrep < Package
   description 'An Ncurses SIP Messages flow viewer'
   homepage 'https://github.com/irontec/sngrep'
-  version '1.4.6'
-  source_url 'https://github.com/irontec/sngrep/releases/download/v1.4.6/sngrep-1.4.6.tar.gz'
+  version '1.4.6-1'
+  source_url 'https://github.com/irontec/sngrep/archive/v1.4.6.tar.gz'
   source_sha256 '638d6557dc68db401b07d73b2e7f8276800281f021fe0c942992566d6b59a48a'
 
   depends_on 'libpcap'
   depends_on 'pcre'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/sngrep-1.4.6-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/sngrep-1.4.6-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/sngrep-1.4.6-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/sngrep-1.4.6-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '09a6ad001c63ce4ac435c148e5bb6df64908b3e06f95520e268510aa89f8c13f',
+     armv7l: '09a6ad001c63ce4ac435c148e5bb6df64908b3e06f95520e268510aa89f8c13f',
+       i686: '9791803d412c9b0f3b25da9fc5c80483aa18262c6411f2eb1b7e636c1ca4c585',
+     x86_64: 'd370d8913abea954dc6420e2dae3d83373917e26935f58c66c38c59809fa275f',
   })
 
   def self.build
-    system 'AUTOCONF_VERSION=2.69 AUTOMAKE_VERSION=1.16 ./bootstrap.sh'
-    system "./configure CPPFLAGS=\"-I#{CREW_PREFIX}/include/ncursesw -I#{CREW_PREFIX}/include/libpcap\" \
+    system './bootstrap.sh'
+    system "./configure CPPFLAGS='-I#{CREW_PREFIX}/include/ncursesw -I#{CREW_PREFIX}/include/pcap' \
           --prefix=#{CREW_PREFIX} \
           --libdir=#{CREW_LIB_PREFIX} \
           --with-openssl \

